GoogleForCreators/web-stories-wp
# Shapes: Add new shapes to library

## Summary

Goes through and normalizes and formats svgs in templates to be normalized clip paths.

## Relevant Technical Choices
Had to add an altered icon path so large shapes with small small stroke widths were visible

## To-do
Go through templates and replace any placeholder pngs with these shapes.

## User-facing changes
Adds new shapes to shapes panel:
![image](https://user-images.githubusercontent.com/40646372/86064096-23786000-ba21-11ea-8482-b5b17587540f.png)

## Testing Instructions
Create a new template or alter and existing one and add all new shapes to the template and make sure they work and are visible.

---



Fixes #1895
